.github/workflows/release.yml

```
@@ -0,0 +1,235 @@
+            </a>
+          </body>
+          </html>' > docs-dist/index.html
+
+      - name: Copy documentation builds
+        run: |
+          cp -r apps/quanta-kit-react-docs/out docs-dist/react || cp -r apps/quanta-kit-react-docs/dist docs-dist/react || echo "React docs not found"
+          cp -r apps/quanta-kit-vue-docs/dist docs-dist/vue || echo "Vue docs not found"
+          cp -r apps/quanta-kit-angular-docs/dist/quanta-kit-angular-docs docs-dist/angular || echo "Angular docs not found"
+
+      - name: Deploy docs to separate GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)